tools-v2/pkg/cli/command/curvefs/warmup/add/add.go

```diff
@@ -170,36 +169,31 @@ func (aCmd *AddCommand) convertFilelist() *cmderror.CmdError {
 	}
 
 	lines := strings.Split(string(data), "\n")
-	validPath := ""
+
 	for _, line := range lines {
 		rel, err := filepath.Rel(aCmd.Mountpoint.MountPoint, line)
-		if err == nil && !strings.HasPrefix(rel, "..") {
-			// convert to curvefs path
-			curvefsAbspath := cobrautil.Path2CurvefsPath(line, aCmd.Mountpoint)
-			validPath += (curvefsAbspath + "\n")
-		} else {
-			convertFail := fmt.Sprintf("[%s] is not saved in curvefs", line)
-			aCmd.ConvertFails = append(aCmd.ConvertFails, convertFail)
+		if err != nil || strings.HasPrefix(rel, "..") {
+			verifyFailMsg := fmt.Sprintf("[%s:%s] is not saved in curvefs", aCmd.Path, line)
+			verifyErr := cmderror.ErrVerifyError()
+			verifyErr.Format(verifyFailMsg, err.Error())
+			break
```

Comment:
You can check all rows and then list them all.

Reply:
done